Extension/src/LanguageServer/settingsPanel.ts

```diff
@@ -252,11 +254,30 @@ export class SettingsPanel {
         }
     }
 
+    private validatePaths(message: any): void {
+        const trimmedPaths = [];
+        for (const value of message.value.split("\n")) {
+            if (fs.existsSync(this.rootUri?.fsPath + path.sep + value.trim())) {

```

Comment:
I don't understand this -- I see rootUri be my workspace folder, but I see value being my workspace folder too, which isn't going to exist.

<img width="494" alt="image" src="https://github.com/microsoft/vscode-cpptools/assets/19859882/b9e84111-11c4-4243-9b96-c26f5c7bfd49">

Reply:
Do you have an example of this code not working?

Reply:
Does the includePath property accept relative paths? I didn't think it does. But either way the path checking shouldn't add on the rootUri path if the path is already a non-relative path.

Reply:
> Does the includePath property accept relative paths?

I was thinking it does

Reply:
An example is ${workspaceFolder} or anything with an absolute path I think.